mne_bids/write.py

```diff
@@ -879,6 +883,14 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
     acquisition, task, run = params['acq'], params['task'], params['run']
     kind = _handle_kind(raw)
 
+    if anonymize:
+        if ext != '.fif':
+            warn('Converting to FIF to anonymize')
+            ext = '.fif'
+        raw.info = anonymize_info(raw.info)
```

Comment:
how will the anonymization work if I have 5 subjects in my BIDS directory? And I want to preserve the longitudinal information.

Reply:
> @alexrockhill should we also add an option to the CLI to anonymize?

Not sure what that is..

Reply:
All the anonymized times have the same relative time, it's just shifted I believe

Reply:
by how many days are they all shifted?

Reply:
Something related to Jan 1, 1970...https://github.com/mne-tools/mne-python/pull/6892

Reply:
daysback : int | None
        Number of days to subtract from all dates.
        If None will default to move date of service to Jan 1 2000

Reply:
Maybe we should expose daysback or just pick a number?

Reply:
If you pick a constant number then it won't be anonymized ...

Reply:
How are you supposed to anonymize and preserve longitudinal structure then?

Reply:
Give it time and think over it. See if the current API makes sense or if it needs an alternate API

Reply:
I think by exposing daysback as long as the person keeps the number that they used anonymous then that works

Reply:
person being user